### DISCUSSION TITLE
Список исключений

### DISCUSSION THREAD

**Discussion by segav247** [qzeleza/kvas#161] [General]
Не нашел возможности добавлять домен в список исключений, чтобы KVAS определенные домены пропускал, и трафик шел не через VPN. Есть такое?

**Comment by segav247** [comment]
Либо как то после обновления таблицы IPSET удалять IP адреса определенных доменов.

**Comment by segav247** [comment]
Проблема в том, что нужные домены (со схожими IP адресами других доменов) попадают в таблицу IPSET и идут через VPN, хотя должны грузиться без него.

**Comment by Ajivitca** [comment]
да, некоторые макаки (большинство криптобирж например) не работают с некоторыми странами и приходится убирать из глобального списка (а если еще и использовать гитхаб списки, то мучение уже). а чекают они по каким - то серверам не своим. типа amazonaws

короче - нужен список для исключения.
